python/test/unit/fem/test_fem_pipeline.py

```diff
@@ -109,3 +111,111 @@ def test_manufactured_poisson(degree, filename, datadir):
     t1 = time.time()
     print("Error assembly time:", t1 - t0)
     assert np.absolute(error) < 1.0e-14
+
+
+@skip_in_parallel
+@pytest.mark.parametrize("filename", ["UnitSquareMesh_triangle.xdmf",
+                                      "UnitCubeMesh_tetra.xdmf",
+                                      # "UnitSquareMesh_quad.xdmf",
+                                      # "UnitCubeMesh_hexahedron.xdmf"
+                                      ])
+@pytest.mark.parametrize("degree", [1])
```

Comment:
The RT and BDM tests share a lot of code.
Why not to parameterize the element family?

Reply:
I think they can all be combined eventually,  but it's still a bit hit-and-miss at the moment which cases are working so it useful to have tests that are green.